### PR TITLE
Chore: Substance Painter Addons for Ayon

### PR DESCRIPTION
**Pull request by moonyuet**
## Changelog Description
Substance Painter Addons for Ayon 
image

## Additional info
n/a

## Testing notes:
1. use the script to create server addon
2. Install your addon into your ayon server
3. If you are in dev mode, create server addon with unzipped and links the directory where you unzipped the addon in ayon_docker
4. Create new bundle with the latest version of application, openpype and substance painter addons.
5. Set Production
6. Go to Project Settings-> Your Project-> Attributes->Applications
7. Add Substance Painter
8. Run ayon launcher with the latest version of addons
9. When you check the launcher in your task you would find substance painter

### CODE REVIEW

**Comment by BigRoy** [comment]
Would also be good to test the **Shelves** feature from project settings as well.

File: server_addon/substancepainter/server/__init__.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+from typing import Type
+
+from ayon_server.addons import BaseServerAddon
+
+from .version import __version__
+from .settings import SubstancePainterSettings, DEFAULT_SPAINTER_SETTINGS
+
+
+class MaxAddon(BaseServerAddon):
```
Comment (iLLiCiTiT) [inline server_addon/substancepainter/server/__init__.py:9]:
```suggestion
class SubstancePainterAddon(BaseServerAddon):
```

**Comment by LiborBatek** [comment]
> Would also be good to test the **Shelves** feature from project settings as well.

Could you elaborate on this more? I dont get what are u refering to. Thx

**Review by LiborBatek — APPROVED** [review]
After configuring all of the settings, SPainter appears in the ```Launcher```

image


Is it correct that I cant see any of ```Ayon tools``` in the host itself? No any trace of it so far...

image

**Comment by moonyuet** [comment]
> After configuring all of the settings, SPainter appears in the `Launcher`
> 
> image
> 
> Is it correct that I cant see any of `Ayon tools` in the host itself? No any trace of it so far...
> 
> image

no it's not correct and it is also expected that there should be the registered host of OP when you launch Substance Painter

**Comment by moonyuet** [comment]
@LiborBatek I updated the commit to use AYON when we are using AYON Launcher.
The screengrab below shows the expected behavior when you launch Substance Painter with AYON.
image

**Pull request merged**